src/lib/phdi-building-blocks/phdi_building_blocks/azure_blob.py

```diff
@@ -5,7 +5,16 @@
 
```

Comment:
I didn't know where else to put this, but it just dawned on me that this module is too specific. I'd like to see this broadened out to include all of our Azure functionality (for now; we can refactor later if necessary). This would include generating credentials, and anything specific to Azure's implementation of the FHIR server. `fhir.py` would then hold code that is generalized to the FHIR API itself.

Reply:
This has been converted to `azure.py`. Thanks, team!